db/urlconnections.go

```diff
@@ -2,6 +2,6 @@ package db
 
 //get from commons
 type ConnectionUrls struct {
-	PgURL string
+	PgURL map[string]string
```

Comment:
Please refer to my comments in the go-btfs-common PR regarding RdURL

Also, why is this not inside db/postgres/? This is entirely Pg urls. Unless you can put redis ones here as well.

Reply:
nvm, you are good.